src/main/pg/rx_spi.c

```diff
@@ -31,10 +31,6 @@
 
 #include "rx/rx_spi.h"
 
-#ifndef RX_SPI_LED_PIN
```

Comment:
This should not be removed.

Reply:
I moved it to src/main/target/common_defaults_post.h, similarly to https://github.com/betaflight/betaflight/pull/7135
I'll open a separate PR for this change.